engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -1038,14 +1048,14 @@ private void checkContainerStatus()
   private AllocateResponse sendContainerAskToRM(List<ContainerRequest> containerRequests, List<ContainerRequest> removedContainerRequests, List<ContainerId> releasedContainers) throws YarnException, IOException
   {
     if (removedContainerRequests.size() > 0) {
-      LOG.info(" Removing container request: " + removedContainerRequests);
+      LOG.info(" Removing container request: (size={}) {}", removedContainerRequests.size(), removedContainerRequests);
```

Comment:
When a collection is a parameter to sl4j logging, all elements of the collection are logged. The size is not a critical piece of info and it is implicitly available anyway.

Remove leading space. It is a typo and is not needed.